subgraph/package.json

```diff
@@ -5,6 +5,8 @@
     "author": "rainlanguage",
     "license": "CAL",
     "dependencies": {
-      "@graphprotocol/graph-ts": "=0.32.0"
+        "@graphprotocol/graph-cli": "^0.56.0",
```

Comment:
this dep ideally wouldn't be here due to graph cli existing in nix, but it has transitive deps that matchstick silently relies on (will error when matchstick tries to run)